docs/src/app/(public)/(content)/react/handbook/forms/page.mdx

```diff
@@ -0,0 +1,606 @@
+# Forms
+
+<Subtitle>A guide to building forms with Base UI components.</Subtitle>
+<Meta name="description" content="A guide to buildling forms with Base UI components." />
+
+Base UI form control components extend the native [constraint validation API](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#the-constraint-validation-api) so you can build forms for collecting user input or providing control over an interface. Integrates seamlessly with third-party libraries like [React Hook Form](#react-hook-form) and [TanStack Form](#tanstack-form).
+
+import { DemoBaseUIForm } from './demos/hero';
+
+<DemoBaseUIForm showExtraPlaygroundLink />
```

Comment:
Select trigger's and radio group's outline is 1px, not 2px like the other controls